### PR TITLE
Python 2 stub for logging

### PR DESCRIPTION
**Pull request by zhangwen0411**

### CODE REVIEW

File: stubs/2.7/logging/__init__.pyi [hunk 1]
```diff
@@ -0,0 +1,236 @@
+# Stubs for logging (Python 3.4)
```
Comment (JukkaL) [inline stubs/2.7/logging/__init__.pyi]: Update the version number in the header

File: stubs/2.7/logging/handlers.pyi [hunk 2]
```diff
@@ -0,0 +1,200 @@
+# Stubs for logging.handlers (Python 3.4)
```
Comment (JukkaL) [inline stubs/2.7/logging/handlers.pyi]: Update the version number (2.7)

**Comment by JukkaL** [comment]
Looks good! Have you tried type checking any example code that user `logging`?

**Comment by zhangwen0411** [comment]
Yeah I tried it out on some files and didn't see any apparent errors...

**Pull request merged**